src/cloudsc2_nl_loki/CMakeLists.txt

```diff
@@ -0,0 +1,197 @@
+# Define this dwarf variant as an ECBuild feature
+ecbuild_add_option( FEATURE CLOUDSC2_NL_LOKI DEFAULT ON
+    DESCRIPTION "Build the Fortran CLOUDSC2 non-linear dwarf with Loki transformations"
+)
+
+function( cloudsc_xmod _TARGET )
+
+    if( TARGET clawfc AND ${LOKI_FRONTEND} STREQUAL "omni" )
```

Comment:
Due to the complexity that OMNI and OFP frontends bring, I'm tempted to say we can exclude them here. We use them to test edge cases in CLOUDSC, but I think there is little gained here from carrying all of that over. Opinions? (Also paging @reuterbal ...)